scania-digital-design-system/sdds
# fix(badges): moving theme rules to component

**Describe pull-request** Moving theming rules to component folder 
**Solving issue** Fixes: [AB#2212](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2212) 
**How to test** 
1. Open badges Storybook of this branch 
3. Open badges Storybook of production 
4. 3. Compare styling- there should not be any difference between those two
